packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -349,7 +349,7 @@ export class SummaryGenerator {
                 // back-compat: cast needed until dep on protocol-definitions version bump
```

Comment:
Can you please check if this comment is still valid (or we can actually remove cast below?)